buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -19,13 +19,13 @@ object Versions {
 
   const val aliyun_sdk_oss: String = "3.4.1" // available: "3.6.0"
 
-  const val aws_java_sdk_core: String = "1.11.271" // available: "1.11.615"
+  const val aws_java_sdk_core: String = "1.11.271" // available: "1.11.618"
```

Comment:
There are some version changes in here. Are they on purpose?

Reply:
I reran the srcVersions plugin and it added the newest version in the comment.

Reply:
I think we should stick to the versions that Flink currently has.